mbedtls/include/README.md

```diff
@@ -0,0 +1,15 @@
+# Toit configuration for mbedtls
+
+This directory contains the configuration for building mbedtls with Toit.
+When compiling for the ESP32 family, the ESP-IDF configuration is used.
```

Comment:
So these two files are ignored on esp-idf? Maybe clarify. Also where is the esp-idf configuration stored?